lib/rester.rb

```diff
@@ -19,7 +29,34 @@ def load_tasks
     def connect(service, params={})
       adapter_opts = Client::Adapters.extract_opts(params)
       adapter = Client::Adapters.connect(service, adapter_opts)
-      Client.new(adapter, params)
+      Client.new(adapter, params).tap { |client| adapter.get('/ping') }
+    end
+
+    def consumer_name
+      @_consumer_name ||= defined?(Rails) ? Rails.application.class.parent_name
+        : "Consumer"
+    end
+
+    def consumer_name=(name)
+      @_consumer_name = name
+    end
```

Comment:
Maybe instead, this can be an option that's passed to the `Client`: `Client.new(name: 'MyService')`.  And it could default to the rails app name or to the service name.